contracts/src/v0.8/dev/ocr2dr/OCR2DRRegistry.sol

```diff
@@ -795,6 +829,13 @@ contract OCR2DRRegistry is
     }
   }
 
+  modifier onlyAuthorizedUsers() {
+    if (!ORACLE.isAuthorizedSender(msg.sender)) {
```

Comment:
In the oracle contract, we authorize users' wallets, while here we check is a contract is authorized. Is this wrong or am I missing something?